test/config/integration_tests.yml

```diff
@@ -48,19 +48,18 @@ testCases:
     kwargs:
       rootpath: /tmp/nnimount/testlocalrootpath
 
-# TODO: Enable this case after nfs server is ready
-#- name: shared-storage-remote-nfs
-#  configFile: test/config/sharedstorage_test/config_sharedstorage_remote_nfs.yml
-#  config:
-#    sharedStorage:
-#      localMountPoint: /tmp/nnimount/testlocalrootpath
-#      remoteMountPoint: /tmp/nnimount/testremoterootpath
-#      nfsServer: $(NFS_IP)
-#      exportedDirectory: /home/nni/mnt/
-#  validator:
-#    class: FileExistValidator
-#    kwargs:
-#      rootpath: /tmp/nnimount/testlocalrootpath
+- name: shared-storage-remote-nfs
+  configFile: test/config/sharedstorage_test/config_sharedstorage_remote_nfs.yml
+  config:
+    sharedStorage:
+      localMountPoint: /tmp/nnimount/testlocalrootpath
```

Comment:
Suggest to set mount point value to pipeline variables, use value in $(LOCAL_MOUNT_POINT) way in case the value will be changed when we reset new path in the future.

Reply:
Changed.